index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Document</title>
+ <link rel="stylesheet" href="./index.css">
+ <link rel="stylesheet" href="./navbar.css">
+</head>
+<body>
+<div id="navbar" >
+  <div id="menu">
+   <a href="#projects" id="profile-link"><p>Projects</p></a>
+  </div>
+  <div id="github">
+   <a href="https://github.com/IsmaelMastronardi" target="_blank"id="profile-link"><p>Github</p></a>
+  </div>
+</div>
```

Comment:
Kindly use HTML semantic tags such as <header> and <nav> for the top section of your webpage